packages/terra-enzyme-intl/tests/jest/__snapshots__/enzyme-intl.test.jsx.snap

```diff
@@ -34,6 +34,7 @@ exports[`mountWithIntl using injectIntl should match the snapshot 1`] = `
     foo="bar"
     intl={
       Object {
+        "__addMessages": [Function],
```

Comment:
I don't know why this snapshot is updated. I'll investigate.

Reply:
My node modules were outdated. clean:install reverted this: https://github.com/cerner/terra-toolkit/pull/583/commits/2251ca53df2fa449f3ced6b1ef0d2d8f1b25e385